nautobot/virtualization/filters.py

```diff
@@ -202,21 +303,57 @@ class VMInterfaceFilterSet(BaseFilterSet, StatusModelFilterSetMixin, CustomField
         to_field_name="name",
         label="Virtual machine",
     )
-    parent_interface_id = django_filters.ModelMultipleChoiceFilter(
+    parent_interface = NaturalKeyOrPKMultipleChoiceFilter(
+        to_field_name="name",
+        queryset=VMInterface.objects.all(),
+        label="Parent interface (name or ID)",
+    )
+    has_parent_interface = RelatedMembershipBooleanFilter(
         field_name="parent_interface",
+        label="Has Parent Interface",
+    )
+    child_interfaces = NaturalKeyOrPKMultipleChoiceFilter(
+        to_field_name="name",
         queryset=VMInterface.objects.all(),
-        label="Parent interface (ID)",
+        label="Child interface (name or ID)",
     )
-    bridge_id = django_filters.ModelMultipleChoiceFilter(
-        field_name="bridge",
+    has_child_interfaces = RelatedMembershipBooleanFilter(
+        field_name="child_interfaces",
+        label="Has Child Interface",
+    )
+    bridge = NaturalKeyOrPKMultipleChoiceFilter(
```

Comment:
On `dcim.interfaces` we're using the reverse name `bridged_interfaces` instead of the forward `bridge`. We should make that here with `bridged_interfaces` and `has_bridged_interfaces`